src/Core/Settings.cpp

```diff
@@ -516,14 +516,15 @@ namespace ErrorCodes
     M(UInt64, max_temporary_data_on_disk_size_for_user, 0, "The maximum amount of data consumed by temporary files on disk in bytes for all concurrently running user queries. Zero means unlimited.", 0)\
     M(UInt64, max_temporary_data_on_disk_size_for_query, 0, "The maximum amount of data consumed by temporary files on disk in bytes for all concurrently running queries. Zero means unlimited.", 0)\
     \
-    M(UInt64, backup_restore_keeper_max_retries, 20, "Max retries for keeper operations during backup or restore", 0) \
+    M(UInt64, backup_restore_keeper_max_retries, 1000, "Max retries for [Zoo]Keeper operations in the middle of a BACKUP or RESTORE operation. Should be big enough so the whole operation won't fail because of a temporary [Zoo]Keeper failure", 0) \
```

Comment:
`20` retries (about 20*backup_restore_keeper_retry_max_backoff_ms = 100 seconds) may be enough for a quick operation, but we don't want a long backup to fail in the middle because of ZooKeeper server was restarting for a couple of minutes.

`1000` retries with default `backup_restore_keeper_retry_max_backoff_ms` gives more than 1 hour which seems to be enough to cover any temporary ZooKeeper failure.